torchvision/transforms/transforms.py

```diff
@@ -1288,13 +1276,7 @@ def get_params(img, scale, ratio, value=0):
         return 0, 0, img_h, img_w, img
 
     def __call__(self, img):
-        """
```

Comment:
Same comment about the docstring

Reply:
Should be addressed in b9a76785c382fac762827ba5a945d03cd56c3711